<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunnerWindow</class>
 <widget class="QWidget" name="RunnerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title">QxRunner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/qxrunner.qrc">
    <normaloff>:/icons/qxrunner_16x16.png</normaloff>:/icons/qxrunner_16x16.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBar" name="runnerToolBar">
     <property name="windowTitle">
      <string comment="@title">Ru&amp;nner Toolbar</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <addaction name="actionStart"/>
     <addaction name="actionStop"/>
     <addaction name="actionSelectAll"/>
     <addaction name="actionUnselectAll"/>
     <addaction name="actionReload"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="runnerFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="Veritas::TestBar" name="progressRun">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="treeRunner">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="spacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelRunText">
           <property name="text">
            <string comment="KDE::DoNotExtract">RanPlaceHolder</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionStart">
   <property name="text">
    <string comment="@action:intoolbar">&amp;Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string comment="@action:intoolbar">S&amp;top</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string comment="@action:intoolbar">Select &amp;All</string>
   </property>
  </action>
  <action name="actionUnselectAll">
   <property name="text">
    <string comment="@action:intoolbar">&amp;Deselect All</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string comment="@action">E&amp;xit</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="text">
    <string comment="@action">&amp;Collapse All</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string comment="@action">&amp;Expand All</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string comment="@action:intoolbar Verb, reload tests">Reload</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Veritas::TestBar</class>
   <extends>QProgressBar</extends>
   <header>internal/runnerwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/qxrunner.qrc"/>
 </resources>
 <connections/>
</ui>
